entity/contents/linear_gradient_contents.cc

```diff
@@ -59,7 +59,7 @@ bool LinearGradientContents::Render(const ContentContext& renderer,
                                    vertices_builder.AppendVertex(vtx);
                                  });
     if (!result) {
-      return false;
+      return true;
```

Comment:
We need to distinguish here whether we failed because the path is empty or we failed because we ran out of memory or something. If we ran out of memory we really need to stop so that the process has as much of a chance as possible to allocate what it can to log a crash etc.

One way to do this would be to create a `TessellationResult` enum, and have it return early in the tessellator if the polyline is empty with an appropriate status.

Reply:
Done. Also added a test unit for the tessellator.